base/applications/mspaint/textedit.cpp

```diff
@@ -197,15 +197,14 @@ void CTextEditWindow::FixEditPos(LPTSTR pszOldText)
 #ifdef NO_GROW_WIDTH
         TEXTMETRIC tm;
         GetTextMetrics(hDC, &tm);
-
-        lstrcat(szText, TEXT("\r\n"));
+        lstrcat(szText, TEXT("x")); // This is a trick to enable the last newlines
```

Comment:
Could you explain what the trick does? (and how appending the "x" does it, while not writing it on screen -- I hope -- ?)

Reply:
`DrawText.DT_CALCRECT` ignores the blank at the end of text.

Reply:
ok :)